.rubocop.yml

```diff
@@ -54,6 +54,7 @@ Metrics/ClassLength:
   - app/controllers/application_controller.rb
   - app/controllers/users/confirmations_controller.rb
   - app/controllers/devise/two_factor_authentication_controller.rb
+  - app/services/idv/session.rb
```

Comment:
I've opened an issue to refactor this later because I wanted to keep this PR as small as possible.